Rettelser (#53)
SpecialeBajs/speciale-rapport: article/Method/remove-alpha-k.tex

\subsection{The effect of $\alpha_k$ in LightGCN}\label{sec:method:alpha-k-effect}
This sections serves to answer the following research question:
\begin{itemize}
	\item \textbf{RQ1}: How does changing $\alpha_k$ affect the performance in LightGCN?
\end{itemize}
We experimented with three different methods to see how changes to how layer combination is performed could effect LightGCN's performance.
This includes removing $\alpha_k$, only using 1 layer and removing the 0th layer embedding.

> Only utilizing 1 layer is required to calculate ALC and BLC to know the performance of each layer.

Reviewer: ```suggestion
Only utilizing 1 layer is required for ALC and BLC to be able to calculate the effect of each layer.
```
